### ISSUE TITLE
3D physics examples

### ISSUE THREAD

**Issue by aglitchman** [defold/examples#53]
Now the project with examples is built as one big game and accordingly all examples have the same project settings. This means that physics is strictly 2D.
How to start adding 3D physics examples? Maybe it's possible to switch to 3d physics with command line arguments, like it's done with bootstrap collection?

**Comment by britzl** [comment]
Yeah, we have talked about splitting up the examples into multiple sub projects to allow for more freedom when creating examples, especially within the render script.

It should be possible to change the  game.project setting though like you suggest though!

**Comment by aglitchman** [comment]
The idea is fail:

![image](https://github.com/user-attachments/assets/a0afa779-dc06-484e-8e46-46dfc607ecc2)

**Comment by aglitchman** [comment]
I was going to add examples like object picking (in 3d), third-person controller, etc. Hm.

I think that 2d and 3d physics is the only thing that can stop the project from building, so it will be enough to split the project with examples into two:
- current project is for 2d examples
- new project is for 3d examples (and move there examples: unlit, screen space etc).

**Comment by aglitchman** [comment]
Related - https://discord.com/channels/250018174974689280/954383800840892426/1278457265292312677

> Examples don't really help since they are separated from the manual and the downloadable examples embed everything at once (it's very hard to find the example collection I'm looking for when everything is in the same project) (and doesn't run in current version of Defold because of some spine example).

Looks like the examples are hard to use when they are in one big project?

**Comment by aglitchman** [comment]
> I'm fine with splitting examples into isolated projects, but I don't have time to do it myself =)))

In fact, it's simple to split everything. The hardest part is modify CI to build examples to separate folders and link together with the website.

**Comment by britzl** [comment]
> > I'm fine with splitting examples into isolated projects, but I don't have time to do it myself =)))
> 
> In fact, it's simple to split everything. The hardest part is modify CI to build examples to separate folders and link together with the website.

I've split the repo into multiple projects and updated the scripts to build defold.com/examples

**Comment by britzl** [comment]
Closing this
